dex/src/main/scala/com/wavesplatform/dex/api/http/routes/MatcherApiRoute.scala

```diff
@@ -272,7 +246,8 @@ class MatcherApiRoute(
     tags = Array("info"),
     response = classOf[String]
   )
-  def getMatcherPublicKey: Route = (pathEndOrSingleSlash & get)(complete(matcherPublicKey.toJson))
+  def getMatcherPublicKey: Route =
+    (pathEndOrSingleSlash & get)(protectedMeasureResponses("getMatcherPublicKeyBase58")(complete(matcherPublicKey.toJson)))
```

Comment:
This looks a bit inconsistent. Why do we need another name for this? It'd be better to have equal names. The idea was to find a slow route and find it using IDE.

Reply:
UP: we will rename methods to better names in a separate task, because this affects tests.